docs/design/layout.md

```diff
@@ -1 +1,427 @@
+
+Margin specifications:
+  * Left:  30px
+  * Top:  30px
+  * Above and below dividers:  30px
+  * Between H2 and HelpText:  10px
+  * After paragraph:  30px
+  * After H4 sub-section:  30px
+
+## The Anchor Solar System
+
+```jsx
+<Spacing>
+  <Horizontal spacing='md' style={{alignItems: 'center'}}>
+    <H2>Phone Numbers</H2>
+    <Anchor icon="viewMore">
+      View all
+    </Anchor>
+    <Anchor icon="msExcel">
+      Export
+    </Anchor>
+    <Anchor icon="copy">
+      Copy to clipboard
+    </Anchor>
+  </Horizontal>
+</Spacing>
+```
+```jsx
+<Spacing>
+  <div style={{display: 'flex', flexDirection: 'row', justifyContent: 'flex-end', alignItems:'center'}}>
+    <div style={{marginRight:'0', marginLeft: '20px'}}>
+      <Anchor icon="copy" >
+        Copy to clipboard
+      </Anchor>
+    </div>
+    <div style={{marginRight:'0', marginLeft: '20px'}}>
+      <Anchor icon="msExcel" style={{marginRight:'0', marginLeft: '20px'}}>
+        Export
+      </Anchor>
+    </div>
+    <div style={{marginRight:'0', marginLeft: '20px'}}>
+      <Anchor icon="viewMore" style={{marginRight:'0', marginLeft: '20px'}}>
+        View all
+      </Anchor>
+    </div>
+    <Button style={{marginRight:'0', marginLeft: '20px'}}>Primary Medium</Button>
+  </div>
+</Spacing>
+```
+Toolbar anchors —usually represented by an icon and uppercase text that’s not underlined- act as actions for a given set of functionality. They generally appear next to a heading, but sometimes may appear next to a submit button depending on the context. Since pages can be subdivided into layout panes, sections, or blocks of content, it can be unclear where in the hierarchy to place these actions.
+
+As much as possible, actions should gravitate to the largest or highest content grouping that makes sense for those actions (“gravitate”, “solar system”… see what we did there). The order of sibling toolbar anchors also follows this idea of 'graviation'. The most utilized anchors should be closest to the heading / button which they are 'orbiting'. In the example above, the order of importance / most utilized is "View All" -> "Export" -> "Copy to Clipboard"
+
+Clicking a toolbar anchor should disable any other sibling anchors. This should be represented by turning the link and icon grey (#666) and setting both to 0.5 opacity. When that function is complete or cancelled, all anchors should transition back to their previous color (likely #008db1) and opacity (1).
+
+## Data Presentation
+
+When displaying input fields, you should aim to present information in a way that users can identify what is currently editable, what was editable, and what was system generated. Use the [Form](/#!/Form), [Flow](/#!/Flow), and [Input](/#!/Input) components to create forms with input fields.
```

Comment:
Flow is deprecated, so ideally we wouldn't be including it in new documentation. We should use Field and Fields instead.